### PR TITLE
Ignore rerun requests for pseudo check

### PR DESCRIPTION
**Pull request by jtmcdole**

### CODE REVIEW

File: app_dart/lib/src/service/scheduler.dart [hunk 1]
```diff
@@ -1126,7 +1126,12 @@ class Scheduler {
         log.fine('Rerun requested by GitHub user: ${checkRunEvent.sender?.login}');
         final String? name = checkRunEvent.checkRun!.name;
         bool success = false;
-        if (name == kCiYamlCheckName) {
+        if (name == kMergeQueueLockName) {
+          final RepositorySlug slug = checkRunEvent.repository!.slug();
+          final int checkSuiteId = checkRunEvent.checkRun!.checkSuite!.id!;
+            log.fine('Requested re-run of "$kMergeQueueLockName" for $slug / $checkSuiteId - ignoring');
```
Comment (yjbanov) [inline app_dart/lib/src/service/scheduler.dart:1132]: dartfmt?

File: app_dart/test/service/scheduler_test.dart [hunk 2]
```diff
@@ -728,9 +803,11 @@ targets:
             'check_suite': <String, dynamic>{'id': 2},
           });
         });
-        final Map<String, dynamic> checkRunEventJson = jsonDecode(checkRunString) as Map<String, dynamic>;
+        final Map<String, dynamic> checkRunEventJson =
+            jsonDecode(checkRunString) as Map<String, dynamic>;
         checkRunEventJson['check_run']['name'] = Scheduler.kCiYamlCheckName;
-        final cocoon_checks.CheckRunEvent checkRunEvent = cocoon_checks.CheckRunEvent.fromJson(checkRunEventJson);
+        final cocoon_checks.CheckRunEvent checkRunEvent =
+            cocoon_checks.CheckRunEvent.fromJson(checkRunEventJson);
```
Comment (yjbanov) [inline app_dart/test/service/scheduler_test.dart:810]: Did you do a 80-column dartfmt?

**Review by yjbanov — APPROVED** [review]
Logic lgtm, but I think dartfmt was misapplied.

File: app_dart/test/service/scheduler_test.dart (hunk 2 above)
Comment (jtmcdole) [reply app_dart/test/service/scheduler_test.dart]: I didn't format correctly. Fixed.

We should totally have a commit check before uploading to save resources.

**Pull request merged**